src/data/sidebars/vue-reactivesearch.yaml

```diff
@@ -35,8 +35,6 @@
               link: /docs/reactivesearch/vue/range/RangeSlider/
             - title: DynamicRangeSlider
               link: /docs/reactivesearch/vue/range/DynamicRangeSlider/
-            - title: RangeInput
-              link: /docs/reactivesearch/vue/range/RangeInput/
       - group: Search Components
```

Comment:
@anik-ghosh-au7 What is reason of removing the `RangeInput` docs link from sidebar?

Reply:
@bietkul I might have mistakenly removed it while resolving the conflicts, I have added it back